apis/resourcecontrollerv2/v1alpha1/register.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Crossplane Authors.
+(C) Copyright IBM Corp. 2020.
```

Comment:
I have used the IBM Copyright throughout, but if there is some reason to use Crossplane's this is easy to change.

Reply:
Since we are contributing to the CNCF Crossplane community project with the CNCF governance I think we should keep the Crossplane's one.